operations/patch_util.go

```diff
@@ -435,9 +430,16 @@ func gitLog(base, ref string) (string, error) {
 }
 
 func gitCommitMessages(base, ref string) (string, error) {
-	args := []string{"--no-show-signature", "--pretty=format:%B", fmt.Sprintf("%s@{upstream}..%s", base, ref)}
+	args := []string{"--no-show-signature", "--pretty=format:%s", "--reverse", fmt.Sprintf("%s@{upstream}..%s", base, ref)}
 	msg, err := gitCmd("log", args...)
-	return strings.Replace(msg, "\n\n", "\n", -1), err // remove multiple line format if applicable
+	if err != nil {
+		return "", errors.Wrap(err, "can't get messages")
+	}
+	// separate multiple commits with <-
+	msg = strings.TrimSpace(msg)
+	msg = strings.Replace(msg, "\n", " <- ", -1)
```

Comment:
Has this been tested with multiple commits? I see how %s might change this, but originally \n\n was used because that's how multiple commits were separated by this command.

Reply:
yes, it was tested.